ironfish/src/rpc/routes/wallet/burnAsset.test.ts

```diff
@@ -83,6 +83,7 @@ describe('burnAsset', () => {
         assetId: assetId.toString('hex'),
         fee: '1',
         value: CurrencyUtils.encode(value),
+        confirmations: 0,
```

Comment:
Do we need to include this in the test? Because the notes can't be burned anymore since we now do confirmation?

Reply:
addressed